app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -422,24 +416,39 @@ public void showVisualizeFragment() {
             // move the play button to the middle
 
             shiftAmnt = (mPlaybackToolbar.getWidth() - mPlayPauseButton.getWidth()) / 2;
-            TranslateAnimation animation = new TranslateAnimation(shiftAmnt, 0, 0, 0);
-            animation.setDuration(200);
-            animation.setInterpolator(new AccelerateDecelerateInterpolator());
-            animation.setFillAfter(true);
-            mPlayPauseButton.setTranslationX(-1 * shiftAmnt);
-            mPlayPauseButton.startAnimation(animation);
-
+            movePlayButton(shiftAmnt);
             // move the info off the screen
-            AlphaAnimation alphaAnimation = new AlphaAnimation(1f, 0f);
-            alphaAnimation.setDuration(300);
-            alphaAnimation.setStartOffset(100);
-            alphaAnimation.setFillAfter(true);
-            alphaAnimation.setInterpolator(new AccelerateInterpolator());
-            findViewById(R.id.ll_show_info).startAnimation(alphaAnimation);
+            moveInfo(1f, 0f);
+
         }
     }
 
     /**
+     * Move the play button to the middle
+     */
+
+    public void movePlayButton(float shiftAmnt) {
+        TranslateAnimation animation = new TranslateAnimation(shiftAmnt, 0, 0, 0);
```

Comment:
What are these 0's here? I don't know what they are and forcing your client to always read documentation to understand random variables in a function isn't good for readability. Extract them as constants and give them proper naming. You might find it helpful to check the documentation of the constructor to see what those numbers are so you can name the constant properly